MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,372 @@
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.RowOrder = 0;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Change the label and editor width
+
+The label and editor width can be set proportionally by using [LabelWidth] and [EditorWidth] properties of [DefaultLayoutSettings] class.
+The [Value] and [UnitType] properties of [DataFormItemLength] each return the value and unit type of the corresponding DataFormItemLength, respectively. 
+
+Label and Editor can be specified with proportionate values such as 0.4* and 0.6* when the [DataFormItemLength.UnitType] is 'Star'.
+Additionally, you can specify a defined width for the editor and label, such as "300, 400," and the [DataFormItemLength.UnitType] is "Absolute."
+
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm">
+        <dataForm:SfDataForm.DefaultLayoutSettings>
+            <dataForm:DataFormDefaultLayoutSettings
+                LabelWidth="0.4*"
+                EditorWidth="0.6*"/>
+        </dataForm:SfDataForm.DefaultLayoutSettings>
+    </dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+        this.dataForm.DefaultLayoutSettings.LabelWidth = new DataFormItemLength(300, DataFormItemLengthUnitType.Absolute);
+        this.dataForm.DefaultLayoutSettings.LabelWidth = new DataFormItemLength(400, DataFormItemLengthUnitType.Absolute);
+
+{% endhighlight %}
+{% highlight C# %}
+
+        this.dataForm.DefaultLayoutSettings.LabelWidth = new DataFormItemLength(0.3, DataFormItemLengthUnitType.Star);
+        this.dataForm.DefaultLayoutSettings.LabelWidth = new DataFormItemLength(0.7, DataFormItemLengthUnitType.Star);
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Label width customization is applicable only when `LabelPosition` is Left.
+
+By default, the available width is divided into 40% for label and 60% for editor.
+
+## Spanning editors rows and columns
+
+Row height and column width can be increased by defining the `DataFormDisplayOptions` attribute or by handling `GenerateDataFormItem` event.
+
+### Row span
+
+#### Using attributes
+
+Row height can be increased by using the [RowSpan] property in the`DataFormDisplayOptions` attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataFormDisplayOptions(RowSpan = 2)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+The row height of each `DataFormItem` can also be increased by using the [RowSpan] property and it will be handled in the `GenerateDataFormItem` event.
```

Comment:
The row height of each `DataFormItem` can also be increased by using the [RowSpan] property, and it will be handled in the `GenerateDataFormItem` event.

Reply:
Content changed